jujugui/static/gui/src/app/models/bundle.js

```diff
@@ -86,7 +86,7 @@ YUI.add('juju-bundle-models', function(Y) {
     */
     parseNameEmail: function(author) {
       var parts = /^([^<]+?) <(.+)>$/.exec(author);
-      if (Y.Lang.isNull(parts)) {
+      if (!parts) {
```

Comment:
technically this would be `parts === null` but I'm not sure that we actually 'need' it to be null here, could you sanity check that for me please.